crates/libxmtp-core/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "libxmtp-core"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
Happy to merge PR as is, but if you're feeling adventurous you can add `vodozemac` here to see how the dependency situation plays out. I could be wrong about conflicts!